src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.ts

```diff
@@ -18,7 +18,9 @@ import {
   FeatureTogglesMap$,
   FeatureTogglesService
 } from '~/app/shared/services/feature-toggles.service';
+import { RefreshIntervalService } from '~/app/shared/services/refresh-interval.service';
 import { SummaryService } from '~/app/shared/services/summary.service';
+import { PgCategoryService } from '../../shared/pg-category.service';
 
 @Component({
```

Comment:
Maybe we need to use the OnPush change detector. This page has a lot of moving pieces and inputs, and probably we don't want to make it over-reactive (in fact, IMHO that should be the default Change Detector in Angular).

Reply:
For now, I went for the current approach. Maybe we can try explore the onPush strategy.